packages/firebase_messaging/android/src/main/java/io/flutter/plugins/firebasemessaging/FirebaseMessagingPlugin.java

```diff
@@ -99,7 +110,28 @@ public void onReceive(Context context, Intent intent) {
 
   @Override
   public void onMethodCall(final MethodCall call, final Result result) {
-    if ("configure".equals(call.method)) {
+    if ("FcmDartService.start".equals(call.method)) {
```

Comment:
For consistency with other plugins that use method names to represent both a class and a method (e.g. [storage](https://github.com/flutter/plugins/blob/master/packages/firebase_storage/android/src/main/java/io/flutter/plugins/firebase/storage/FirebaseStoragePlugin.java#L67)), consider using `#` here.

Also if the class you're calling is `FlutterFirebaseMessagingService` would it make sense to put that here as well?

Reply:
Used `#` for consistency. 

`FlutterFIrebaseMessagingService` starts without the call from the Dart side. I'm happy to change the name but it should be something that indicates the setting up of the background channel.